### DISCUSSION TITLE
croping TIF image

### DISCUSSION THREAD

**Discussion by SylvainArd** [ImageMagick/ImageMagick#5104] [Help]
Hi !
what is the command for cropping a TIF image like this : 
![Capture](https://user-images.githubusercontent.com/19763527/166117604-0b9a8bd4-02b1-4b17-85b5-6349d9a21fd0.PNG)
Thank you
Best regards

**Comment by fmw42** [comment]
-crop WxH+X+Y +repage

See https://legacy.imagemagick.org/Usage/crop/#crop

**Reply by fmw42** [reply]
For novices, see

http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=9620
http://www.imagemagick.org/script/command-line-processing.php
https://imagemagick.org/script/command-line-options.php
http://www.imagemagick.org/Usage/reference.html
http://www.imagemagick.org/Usage/
https://github.com/ImageMagick/usage-markdownhttps://imagemagick.org/script/porting.php#cli
https://legacy.imagemagick.org/discourse-server/index.php

**Comment by SylvainArd** [comment]
ok thank you I succeeded by magick mire057.tif -crop 30x40+100+200 +repage
out.tif
Sylvain Ard
0549507724
0778380991
***@***.***
http://sylvain-ard.fr
Entreprise individuelle SIRET : 80079243400022
Appt 26 Bât A Résidence Le Patio
83 rue de la Bugellerie
86000 Poitiers


Le sam. 30 avr. 2022 à 20:32, Fred Weinhaus ***@***.***> a
écrit :
